docker-bake.hcl

```
@@ -100,6 +100,9 @@ target "_common_docs" {
 }
 
 target "docs-export" {
+  args = {
+    DOCS_BASEURL = "/"
+  }
```

Related to my previous comment https://github.com/distribution/distribution/pull/4102/files#r1359480999:

```suggestion
variable DOCS_BASEURL" {
  default = null
}

target "docs-export" {
  args = {
    DOCS_BASEURL = DOCS_BASEURL
  }
```

More info about `null` val in target args: https://docs.docker.com/build/bake/reference/#targetargs